src/main/java/com/google/cloudassets/discovery/exceptions/ConfigTableException.java

```diff
@@ -0,0 +1,11 @@
+package com.google.cloudassets.discovery.exceptions;
```

Comment:
In order to save some work and reduce boilerplate, you can create a class called "AssetDiscoveryExceptions" and put all these exception classes as static nested classes.

Reply:
Good idea, will add it to the PR in which we will refactor the use of the error messages to use fluent logger string parsing